API-strategie-modulen/API-strategie-mod-geo/crs.md

```diff
@@ -138,15 +138,16 @@ The CRS can be specified for request and response individually using parameters
 In an API that supports the creation of items, POST requests with geospatial content in the body may be sent by a client to the server. In that case, it is necessary to indicate the CRS used, unless CRS84, the default CRS, is used.
```

Comment:
Ook aanpassen of verwijderen?

Reply:
aangepast, zie [9e878ba](https://github.com/Geonovum/KP-APIs/pull/570/commits/9e878ba711f1a6c35f212dca696222d8b26c229d)